packages/steps/src/__tests__/fixtures/build.yml

```
@@ -29,3 +29,8 @@ build:
         name: Use non-default shell
         shell: /nib/hsab
         command: echo 123
+    - eas/save-cache:
+        inputs:
+          key: cache-key
+          paths:
+            - src
```

nit: new line at the end of file. (missing code editor setting on your side?)

```suggestion
            - src

```